indigo/analysis/refs/base.py

```diff
@@ -246,25 +240,50 @@ class SectionRefsFinderENG(BaseSectionRefsFinder):
         TODO: match paragraphs
         TODO: match multiple sections
         TODO: match ranges of sections
-
     """
 
     # country, language, locality
     locale = (None, 'eng', None)
 
-    section_re = re.compile(
-        r'''(?P<ref>
-        (?P<section_ref>
-        \b[sS]ections?\s+
-        (?P<num>\d+[A-Z]*)
+    ref_re = re.compile(
+        r'''
+        (?P<ref>
+          \b[sS]ections?\s+
+          (?P<num>\d+[A-Z]*)
         )
         (?P<subsection_ref>\s*\(\d+[A-Z]*\))?
         (?P<paragraph_ref>\s*\([a-z]+[A-Z]*\))?
         (?P<subparagraph_ref>\s*\([ivx]+[A-Z]*\))?
         (?P<item_ref>\s*\([a-z]{2,}[A-Z]*\))?
         (?!\s*\()
         (\s+of\s+(this\s+Act|the\s+|Act\s+)?)?
-        )''',
+        ''',
         re.X)
 
     candidate_xpath = ".//text()[contains(., 'section') and not(ancestor::a:ref)]"
```

Comment:
Worth noting that xpath is case sensitive, so this will never find text elements with "Section".

Reply:
thanks! 
```suggestion
     candidate_xpath = ".//text()[contains(translate(., 'S', 's'), 'section') and not(ancestor::a:ref)]"
```